test/node/user-agent.js

```diff
@@ -6,7 +6,7 @@ const base = setup.uri;
 
 describe("req.get()", () => {
   it("should set a default user-agent", () =>
-    request.get(`${base}/ua`).then(res => {
+    request.put(`${base}/ua`).then(res => {
```

Comment:
Can you explain this change?

Reply:
http/2 client module cannot send body with head or get Method. But, it seems to not need here. I check this again later.

Reply:
I test locally and noticed that this change is not needed. I will revert this change.